keadm/cmd/keadm/app/cmd/cloud/init.go

```diff
@@ -121,23 +121,25 @@ func addForceOptionsFlags(cmd *cobra.Command, initOpts *types.InitOptions) {
 //AddInit2ToolsList reads the flagData (containing val and default val) and join options to fill the list of tools.
 func AddInit2ToolsList(toolList map[string]types.ToolsInstaller, initOpts *types.InitOptions) error {
 	var latestVersion string
-	var kubeedgeVersion string
-	for i := 0; i < util.RetryTimes; i++ {
-		version, err := util.GetLatestVersion()
-		if err != nil {
-			fmt.Println("Failed to get the latest KubeEdge release version, error: ", err)
-			continue
+	kubeedgeVersion := initOpts.KubeEdgeVersion
+	if kubeedgeVersion == "" {
```

Comment:
LGTM
this is right, if we've already specified the version, there's no need to check the latest release version